examples/next/pages/ui/components/account-settings/delete-user/aws-exports.js

```diff
@@ -0,0 +1,2 @@
+import awsExports from '@environments/auth/auth-with-email/src/aws-exports';
```

Comment:
Added example just for internal development purposes. E2e specs will follow with more complete example.